src/systems/ec/implementations/react/templates/pages/job-person-list/src/PersonListPage.jsx

```diff
@@ -228,181 +229,173 @@ const PersonListPage = ({ siteHeader, footer, template }) => (
-                style={{
-                  backgroundImage:
-                    'url("https://upload.wikimedia.org/wikipedia/commons/thumb/2/27/Emojione_1F464.svg/1024px-Emojione_1F464.svg.png")',
-                  backgroundSize: 'contain',
-                }}
-              />
-              <div className="ecl-u-flex-grow-1">
-                <h3 className="ecl-u-type-heading-3 ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-none ecl-u-mb-xs">
-                  Nathalie Chaze
-                </h3>
-                <p className="ecl-u-type-paragraph ecl-u-type-uppercase ecl-u-type-color-grey-75 ecl-u-mt-none ecl-u-type-s">
-                  Deputy Head of Cabinet
-                </p>
-                <DescriptionList variant="horizontal">
-                  <DescriptionTerm>Phone number</DescriptionTerm>
-                  <DescriptionDefinition>+32 2 29 64816</DescriptionDefinition>
-                  <DescriptionTerm>Responsibilities</DescriptionTerm>
-                  <DescriptionDefinition>
-                    <UnorderedList className="ecl-u-mv-m">
-                      <UnorderedListItem>
-                        Food information and composition
-                      </UnorderedListItem>
-                      <UnorderedListItem>
-                        Food processing technologies and novel foods
-                      </UnorderedListItem>
-                    </UnorderedList>
-                    <Link
-                      variant="standalone"
-                      label="See all responsibilities"
-                      href="/example"
-                      icon={{
-                        shape: 'ui--corner-arrow',
-                        size: 'xs',
-                        transform: 'rotate-90',
-                      }}
-                    />
-                  </DescriptionDefinition>
-                </DescriptionList>
-              </div>
-            </article>
+            <ContentItem
+              images={{
+                position: 'left',
+                desktop: {
+                  alt: 'Example image',
+                  src:
+                    'https://inno-ecl.s3.amazonaws.com/media/examples/example-person.png',
+                },
+              }}
+            >
+              <h3 className="ecl-u-type-heading-3 ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-none ecl-u-mb-xs">
+                Nathalie Chaze
+              </h3>
+              <p className="ecl-u-type-paragraph ecl-u-type-uppercase ecl-u-type-color-grey-75 ecl-u-mt-none ecl-u-type-s">
+                Deputy Head of Cabinet
+              </p>
+              <DescriptionList variant="horizontal">
+                <DescriptionTerm>Phone number</DescriptionTerm>
+                <DescriptionDefinition>+32 2 29 64816</DescriptionDefinition>
+                <DescriptionTerm>Responsibilities</DescriptionTerm>
+                <DescriptionDefinition>
+                  <UnorderedList className="ecl-u-mv-m">
```

Comment:
This is already the case, but here we must remove top margin that we don't have on the previous and next row's lists. Should be replaced by `ecl-u-mb-m`

On all the lists, I would also force left padding to better align it with the content: `ecl-u-pl-m`